src/preprocess/vhi.py

```diff
@@ -0,0 +1,173 @@
+"""
+
+- Add lat lon coordinates
+- add time coordinates
+- subset Kenya
+- merge into one time (~500MB)
+"""
+from pathlib import Path
+import pathlib
+import xarray as xr
+import multiprocessing
+from typing import List, Optional, Tuple, Union
+import pickle
+from functools import partial
+
+from xarray import Dataset
+
+from .base import (BasePreProcessor, get_kenya)
+from .preprocess_vhi import (
```

Comment:
Since we have two files dedicated to `vhi` preprocessing, can we move them into their own folder?

Reply:
(Or just make them part of the preprocessing class)